test/unit/checkbox_question_presenter_test.rb

```diff
@@ -15,6 +15,7 @@ class CheckboxQuestionPresenterTest < ActiveSupport::TestCase
       @renderer.stubs(:option).with(:option3).returns({ label: "Option 3" })
 
       @presenter = CheckboxQuestionPresenter.new(@question, nil, nil, renderer: @renderer)
+      @presenter.stubs(:response_for_current_question).returns(nil)
```

Comment:
It might be useful to explain why this was added in the commit message, I had to do a bit of digging to understand why this line was added.